ansible/playbooks/hosts

```diff
@@ -47,11 +47,11 @@ apollo-monitor.genome.edu.au
 apollo-portal.genome.edu.au
 #apollo-portal-main.genome.edu.au # use this when apollo-portal redirects to apollo-portal-offline
 apollo-deploy.genome.edu.au
-ubuntu22-test.genome.edu.au
 ubuntu20-test.genome.edu.au
 #ubuntu18-test.genome.edu.au
 
-
+[dev_portal_webservers]
+ubuntu22-test.genome.edu.au  ansible_connection=ssh  ansible_user=ubuntu
 
```

Comment:
this should go in it's own inventory file. See `ansible/playbooks/buildmonitor.inventory` for example.

Reply:
also it should be defined as `newportalvms` to match the group_vars.